packages/dex2jar/build.sh

```diff
@@ -0,0 +1,22 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/pxb1988/dex2jar
+TERMUX_PKG_DESCRIPTION="Tools to work with android .dex and java .class files"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=2.2
+TERMUX_PKG_SRCURL=https://github.com/pxb1988/dex2jar/releases/download/v2.2-SNAPSHOT-2021-10-31/dex-tools-${TERMUX_PKG_VERSION}-SNAPSHOT-2021-10-31.zip
```

Comment:
Since this is a pre-release of 2.2 the version should be lower than 2.2, and indicate that it is a snapshot in some way. Version `2.1.2021.10.31` would work for example